package.json

```diff
@@ -28,6 +29,7 @@
     "cookie-parser": "^1.4.4",
     "crypto-js": "3.1.9-1",
     "csurf": "^1.10.0",
+    "cypress": "^6.2.0",
```

Comment:
1. You forgot to commit `npm-shrinkwrap.json` changes, as builds relay on that file as well, since dependencies in `package.json` have to be in sync with `npm-shrinkwrap.json` that is why build failed
2. Why this is not in `devDependencies` ?